app/workers/deletes.php

```diff
@@ -225,7 +225,7 @@ protected function deleteUsageStats(string $datetime1d, string $datetime30m)
             ], $dbForProject);
 
             $this->deleteByGroup('stats', [
-                Query::lessThan('time', [$datetime30m]),
+                Query::lessThan('time', $datetime30m),
```

Comment:
This doesn't make sense? being array or not it should be same right? we have the same below `Query::equal('period', ['30m']`?

However, the change should not make a difference as far as I know how database query works

Reply:
The DateTime validator was throwing errors.
The diff is period is a string  while time is dateTime attr